src-common/vcastServer.ts

```diff
@@ -34,7 +34,7 @@ export interface clientRequestType {
   options?: string;
 }
 
-function serverURL() {
+export function serverURL() {
```

Comment:
Denis could you please add comments for these functions that are newly exported to indicate that they are being exported for testing purposes please.

Ping me after this and we can merge this.